.clippy.toml

```diff
@@ -1,3 +1,4 @@
+type-complexity-threshold = 999
```

Comment:
😂

Reply:
As Bodil would say "I have the power of [Oleg](https://okmij.org/ftp/)!"